test-network-function/platform/suite.go

```diff
@@ -148,9 +148,10 @@ func getPodNodeName(context *interactive.Context, podName, podNamespace string)
 	return podNameTester.GetNodeName()
 }
 
-func getCurrentKernelCmdlineArgs(targetPodOc *interactive.Oc) map[string]string {
+func getCurrentKernelCmdlineArgs(context *interactive.Context) map[string]string {
 	currentKernelCmdlineArgsTester := currentkernelcmdlineargs.NewCurrentKernelCmdlineArgs(common.DefaultTimeout)
-	test, err := tnf.NewTest(targetPodOc.GetExpecter(), currentKernelCmdlineArgsTester, []reel.Handler{currentKernelCmdlineArgsTester}, targetPodOc.GetErrorChannel())
+
+	test, err := tnf.NewTest(context.GetExpecter(), currentKernelCmdlineArgsTester, []reel.Handler{currentKernelCmdlineArgsTester}, context.GetErrorChannel())
```

Comment:
This doesn't look right. The "cat /proc/cmdline" command shouldn't be executed locally.